FWCore/Framework/interface/produce_helpers.h

```diff
@@ -23,88 +23,85 @@
 #include <optional>
 // user include files
 
-// forward declarations
-namespace edm {
-   namespace eventsetup {
-      
-      namespace produce { struct Null;}
-      
-     template<typename FromT, typename ToT> void moveFromTo(FromT& iFrom,
-                                                            ToT & iTo) {
-       iTo = std::move(iFrom);
-     }
+namespace edm::eventsetup {
```

Comment:
Was this just a change from 3 spaces to 2 ? If so, please revert.

Reply:
Namespace collapsing was the change.